README.md

```diff
@@ -58,11 +72,9 @@ BROADCAST_DRIVER=sns
 
 Finally, don't forget to enable the [Broadcast Service Provider](https://laravel.com/docs/master/broadcasting#broadcast-service-provider).
 
-## Usage
-
-Here we will simply re-use the power of Laravel Broadcasting, out of the box, with some minor additional functionalities for the Eloquent Model Events.
+---
 
-### Basic Events
+### Usage: Basic Events
```

Comment:
There is a lot of duplication between this section and Usage: Model Broadcasting

I think we need to reorganise this as it feels like I am reading the same stuff again.

Reply:
Myeah I agree...